app/javascript/tubes-to-rack/components/TubesToRack.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <lb-page>
+    <lb-loading-modal
+      v-if="loading"
+      :message="progressMessage"
+    />
+    <lb-main-content>
+      <b-card
+        bg-variant="dark"
+        text-variant="white"
+      >
+        <h3>Tube rack UI goes here</h3>
+        <p>
+          Scanned tubes will appear in a UI representation of the tube rack.
+          This will be similar in function to the plate UI but will be capable
+          of opening the tube information page when viewed after creation.
+        </p>
+      </b-card>
+    </lb-main-content>
+    <lb-sidebar>
+      <b-card
+        header="Scan tubes"
+        header-tag="h3"
+      >
+        <b-form-group
+          label="Scan the tube barcodes into the relevant rack coordinates:"
+          style="position:relative; height:460px; overflow-y:scroll;"
+        >
+          <lb-labware-scan
+            v-for="i in tubeCount"
+            :key="i"
+            :api="devourApi"
+            :label="indexToName(i - 1)"
+            :fields="tubeFields"
+            :includes="tubeIncludes"
+            :validators="scanValidators"
+            :colour-index="i"
+            :labware-type="'tube'"
+            :valid-message="''"
```

Comment:
Huh, so the valid message is getting passed in as a prop... (I've been jumping about a bit in this review, so sorry if this comment appears before the one its a reference to)